NuGet.Config

```diff
@@ -5,5 +5,6 @@
     <add key="Local" value="tools/LocalNugetFeed" />
     <add key="roslyn" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
     <add key="azure-sdk-tools" value="https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json" />
+    <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
```

Comment:
Should we wait until they publish this package to nuget.org?

Reply:
Yes, I'm not merging this with a custom feed in.